docs/documenting-schema.rst

```diff
@@ -0,0 +1,73 @@
+.. _documenting-schema:
+
+Documenting schema
+=======================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
+
+Keeping your documentation close to the code that supports it is a great way to ensure that it is always accurate and up-to-date. To support this workflow, GraphQL allows programmers to write descriptions directly alongside types in `GraphQL Schema Definition Language syntax <https://facebook.github.io/graphql/June2018/#sec-Descriptions>`_.
+
+
+Writing Descriptions
+--------------------
+
+GraphQL Descriptions are declared using a python-like docstring format, and may include Markdown features::
```

Comment:
Looks odd.
```suggestion
GraphQL descriptions are declared using a python-like docstring format, and can include Markdown features::
```

Reply:
I think you're right here.  I'll lower-case "descriptions" throughout the page.